major_event_log/models.py

```diff
@@ -32,13 +32,13 @@ class Event(models.Model):
     contact_email = models.EmailField()
 
     def get_absolute_url(self):
-        return(reverse('major-event-log:event_details', args=[self.id]))
+        return (reverse('major-event-log:event_details', args=[self.id]))
```

Comment:
Do we need the parentheses around the call to `reverse` or could those go away?

Reply:
We don't need them. I will remove them.